include/phasar/Utils/NlohmannLogging.h

```diff
@@ -0,0 +1,59 @@
+/******************************************************************************
+ * Copyright (c) 2022 Martin Mory.
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of LICENSE.txt.
+ *
+ * Contributors:
+ *     Martin Mory and others
+ *****************************************************************************/
+
+#ifndef PHASAR_UTILS_NLOHMANNLOGGING_H
+#define PHASAR_UTILS_NLOHMANNLOGGING_H
+
+#include "llvm/Support/FormatVariadic.h"
+
+#include "nlohmann/json.hpp"
+
+namespace llvm {
+class raw_ostream;
+}
+
+namespace nlohmann {
+namespace detail {
+template <typename CharType>
```

Comment:
Does this really need to be a template? As far as I know, llvm::raw_ostream only supports one CharType - `char`

Reply:
removed.